### PR TITLE
Added domino-cli latest

### PR DESCRIPTION
**Pull request by hakamadare**
Installer for Domino Client (https://www.dominodatalab.com/)

### CODE REVIEW

**Comment by hakamadare** [comment]
nb: this cask doesn't work when used with a Homebrew installed somewhere other than `/usr/local`.  I use Boxen to install Homebrew, and my cask fails installation (unsurprisingly) with:

```
==> Symlinking Binary 'domino' to '/usr/local/bin/domino'
==> ln: /usr/local/bin/domino: Permission denied
Error: Command failed to execute!

==> Failed command:
["/bin/ln", "-hfs", "--", "#<Pathname:/Applications/domino/domino>", "#<Pathname:/usr/local/bin/domino>"]
```

I haven't had this problem with other casks; is there a standard workaround for this problem?

File: Casks/domino-cli.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  homepage 'https://www.dominodatalab.com/'
+  license :unknown
+
+  auto_updates true
+
+  installer script: 'Domino Installer.app/Contents/MacOS/JavaApplicationStub',
+            sudo:   true
+  binary '/Applications/domino/domino'
+
+  uninstall script: '/Applications/domino/Domino Uninstaller.app/Contents/MacOS/JavaApplicationStub'
```
Comment (vitorgalvao) [inline Casks/domino-cli.rb]: Won’t this call a GUI? If so, we can’t accept it.

[Scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md) `uninstall`.

File: Casks/domino-cli.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  name 'Domino Client'
+  name 'Domino CLI'
+  homepage 'https://www.dominodatalab.com/'
+  license :unknown
+
+  auto_updates true
+
+  installer script: 'Domino Installer.app/Contents/MacOS/JavaApplicationStub',
+            sudo:   true
+  binary '/Applications/domino/domino'
```
Comment (vitorgalvao) [inline Casks/domino-cli.rb]: Is there anywhere on the site where they recommend this? If it is not expected behaviour, we will not include it.

File: Casks/domino-cli.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+cask 'domino-cli' do
+  version :latest
+  sha256 :no_check
+
+  # app.dominodatalab.com is the official download host per the vendor support page
+  # http://support.dominodatalab.com/hc/en-us/articles/204856475-Installing-the-Domino-Client-CLI-
+  url 'https://app.dominodatalab.com/assets/cli/default/domino-install-mac.dmg'
+  name 'Domino Client'
```
Comment (vitorgalvao) [inline Casks/domino-cli.rb]: Make it just one: `name 'Domino CLI Client'`.

File: Casks/domino-cli.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+cask 'domino-cli' do
+  version :latest
+  sha256 :no_check
+
+  # app.dominodatalab.com is the official download host per the vendor support page
```
Comment (vitorgalvao) [inline Casks/domino-cli.rb]: Remove the comments. Not needed.

**Comment by vitorgalvao** [comment]
There was a [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates) when you submitted. It must be filled.

If you were not presented a template, you may have submitted via a CLI tool (likely [`hub`](https://hub.github.com/)). It that was the case, you may wish to see if the tool has a way to let you use the template (`hub` [is working on it](https://github.com/github/hub/issues/1118)).

In either case, please [refer to the template](https://raw.githubusercontent.com/caskroom/homebrew-cask/master/.github/PULL_REQUEST_TEMPLATE.md) and update your comment.

**Pull request closed without merging**